mimic/model/nova_objects.py

```diff
@@ -815,6 +816,38 @@ def request_action(self, http_action_request, server_id):
         else:
             return dumps(bad_request("There is no such action currently supported", http_action_request))
 
+    def _create_random_list_of_flavors(self):
+        """
+        Creates a random list of 10 flavors.
+        """
+        for each in range(10):
+            flavor_id = choice([str(each), "onmetal-" + str(each),
+                                "mimic" + str(each)])
+            flavor_name = "{0}GB Instance".format(flavor_id)
+            ram = choice([32768, 131072, 524288, 65536,
```

Comment:
I think this is fine, but be aware that OnMetalScout in Arsenal uses amount of ram as a unique key at the moment. So if there are two flavors with the same amount of ram, they will look identical to Arsenal. I plan to update this soon to key off of the flavor metadata added to nodes by @jimrollenhagen.